Apps/Sandcastle/index.html

```diff
@@ -21,6 +21,7 @@
     <script src="jsHintOptions.js"></script>
     <script src="gallery/gallery-index.js"></script>
     <script src="CesiumSandcastle.js"></script>
+    <script src="../../Source/Cesium.js"></script>
```

Comment:
Indeed including Source/Cesium.js again was redundant. Thanks for pointing that out @emackey. Fixed, @pjcozzi